pkg/reconciler/internal/conditions/conditions.go

```diff
@@ -28,7 +28,9 @@ const (
 	TypeInitialized    = "Initialized"
 	TypeDeployed       = "Deployed"
 	TypeReleaseFailed  = "ReleaseFailed"
+	TypeRollbackFailed = "TypeRollbackFailed"
```

Comment:
```suggestion
	TypeRollbackFailed = "RollbackFailed"
```
None of the other types starts with `Type`...